xhype/xhype/src/hv/mod.rs

```diff
@@ -0,0 +1,613 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
did you write this or did you get it from somewhere else?

Reply:
The three files mod.rs, ffi.rs, and vmx.rs are based on the code in https://github.com/RWTH-OS/xhypervisor. I modified and added some other functions.